static/js/place/chart.tsx

```diff
@@ -161,7 +164,8 @@ class Chart extends React.Component<ChartPropType, ChartStateType> {
     }
     if (
       this.props.chartType === chartTypeEnum.CHOROPLETH &&
-      (!this.state.choroplethDataGroup ||
+      (_.isEmpty(this.state.choroplethDataGroup) ||
```

Comment:
move this check to 443 (i.e. don't set state if these conditions hold, and set state.display = false